docs/generate-time-series/go.rst

```diff
@@ -0,0 +1,374 @@
+
+.. code-block:: console
+
+    $ go mod init example.com/time-series-go
+
+You should see a `go.mod` file in the current directory with contents similar to:
+
+.. code-block:: console
+
+    module example.com/time-series-go
+
+    go 1.14
+
+Next, create a file named `main.go` in the same directory:
+
+.. code-block:: console
+
+   $ touch main.go
+
+Open up your favorite code editor. We will be editing this file a lot!
+
+
+Get ISS telemetry data
+======================
+
+You can get telemetry data from `Open Notify`_, a third-party service that
+provides a simple API to consume data from NASA (specifically, the current
+location of the International Space Station). The endpoint for this data is
+`<http://api.open-notify.org/iss-now.json>`_.
+
+
+In your `main.go` file, declare the main package at the top (to tell the 
+compiler that our program is an executable), import some packages from the 
+`standard library`_ that will be used in this tutorial and declare a main 
+function which will be the entry point of our executable program:
+
+.. code-block:: js
+
+    package main
+
+    import (
+        "encoding/json"
+        "fmt"
+        "io/ioutil"
+        "log"
+        "net/http"
+    )
+
+    func main() {
+
+    }
+
+Then, look at the JSON data that gets returned from going to the Open
+Notify API endpoint at `<http://api.open-notify.org/iss-now.json>`_. The 
+endpoint returns a JSON payload, which contains an ``iss_position`` object
+with ``latitude`` and ``longitude`` data.
+
+.. code-block:: js
+
+   {
+        "message":"success",
+        "timestamp":1591703638,
+        "iss_position":{
+            "longitude":"84.9504",
+            "latitude":"41.6582"
+        }
+    }
+
+The longitude and latitude of the International Space Station changes 
+constantly and is what you want to extract from this payload and insert into
+CrateDB. To parse this JSON, you can create a `struct`_ that can be used to 
+`unmarshal`_ the data into. When you unmarshal JSON into a struct, the 
+function matches incoming object keys to the keys in the struct field name
+or its tag. By default, object keys which don't have a corresponding struct 
+field are ignored.
+
+.. code-block:: js
+
+    type issInfo struct {
+        IssPosition struct {
+            Longitude string `json:"longitude"`
+            Latitude  string `json:"latitude"`
+        } `json:"iss_position"`
+    }
+
+Now, create a function that makes an HTTP GET request to the Open Notify API 
+endpoint and returns longitude and latitude as a `geo_point`_ declaration.
+
+The Go standard library comes with great support for HTTP client and server
+implementations in the `net/http`_ package. In this example you’ll use it to 
+issue an HTTP GET request to the API endpoint. 
+
+Then you will read the response body and unmarshal the JSON into your defined 
+struct ``issInfo``. 
+
+Lastly, you will format the return string and return it. 
```

Comment:
same as above

I would reword this:

"Lastly, you can format the return string and return it."